src/web_app_skeleton/src/actions/api_action.cr.ecr

```diff
@@ -0,0 +1,18 @@
+abstract class ApiAction < Lucky::Action
+  <%- if generate_auth? -%>
+  # By default all actions require sign in, unless you use the
+  # `Auth::SkipRequireSignIn` module in `src/mixins/auth/skip_require_sign_in.cr`
+  <%- if api_only? -%>
+  include Auth::Api::RequireSignIn
+  <%- else -%>
+  include Auth::RequireSignIn
+  <%- end -%>
```

Comment:
Since this is an ApiAction I don't think it needs the the `api_only?` conditional part. I think it should always use `Auth::Api::RequireSignIn`

Reply:
So that means that browser apps would need to create a separate `RequireSignIn` mixin for their api actions, instead of using the one that is generated for them right?

Reply:
Hmm good point. There's what I think this should be:

```crystal
<%- if generate_auth? -%>
include Auth::Api::RequireSignIn
<%- end %> 
```

And the `else` should be omitted. The ApiAction is always created, so the only conditional would be whether it should generate auth or not. So that means the `Auth::ApiRequireSignIn` module should be created for full or api_only apps, but only if generated_auth? is true

Reply:
Yes that makes sense